### PR TITLE
migrated to native fetch

### PR DESCRIPTION
**Pull request by AlexGalichenko**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -20,7 +15,7 @@ export async function dataTable2Object(dataTable: DataTable): Promise<{ [key: st
 }
 
 export async function sendHttpRequest(requestUrl: string, conf: RequestInit, context?: IWorld): Promise<Response> {
-  const response = await originalSendHttpRequest(requestUrl, conf);
+  const response = await fetch(requestUrl, conf);
```
Comment (KirillBogdanets) [inline src/utils.ts:18]: maybe I'm missing something but where did you take `fetch` ?

Comment (KirillBogdanets) [reply src/utils.ts:18]: ohh, it is native module, my bad

File: src/websocket.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    const regexp = typeof regexpTemplate === 'string' ? new RegExp(regexpTemplate, 'gmi') : regexpTemplate;
+    const message = await new Promise(resolve => {
+        ws.on('message', (message => {
+            if (regexp.test(message)) resolve(message);
+        }));
+    });
+    memory.setValue(memoryKey, message);
+});
+
+async function sendMessage({messageKey, wsKey}: {messageKey: string, wsKey: string}) {
```
Comment (KirillBogdanets) [inline src/websocket.ts:34]: maybe it will be a good idea to move it to the `utils.ts` module?

Comment (KirillBogdanets) [reply src/websocket.ts:34]: agree

**Review by KirillBogdanets — APPROVED** [review]

**Review by nomid0612 — APPROVED** [review]

**Pull request merged**